guides/m1x/index.html

```diff
@@ -4,8 +4,46 @@
 ---
 
 <p>Welcome to the home page for Magento 1.x documentation for installation, configuration, developers, patches, and more. Here you'll find articles that were formerly located on <code>http://magentocommerce.com/knowledge-base</code>.</p>
-<p>For information you don't find here&mdash;including Release Notes&mdash;see <a href="http://magento.com/help/documentation" target="_blank">the User Guide page</a>.</p>
+<p>For information you don't find here&mdash;including Release Notes&mdash;see <a href="https://magento.com/resources/technical" target="_blank">the User Guide page</a>.</p>
 
+<div class="toc">
```

Comment:
This is a duplicate of the TOC. Please remove it.

Reply:
@jeff-matthews I have fixed this issue.